packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/vault/models/EncryptRequest.java

```diff
@@ -0,0 +1,33 @@
+package cloud.pangeacyber.pangea.vault.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class EncryptRequest {
+    @JsonProperty("id")
+    String id;
+
+    @JsonProperty("plain_text")
+    String plainText;
+
+    public EncryptRequest(String id, String plainText) {
+        this.id = id;
+        this.plainText = plainText;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getPlainText() {
+        return plainText;
+    }
+
+    public void setPlainText(String plainText) {
```

Comment:
don't you think we should have an option to receive the plain buffer and encoding ourselves, instead of always requesting the data in base64? I'm not sure, it's a question. Same for verify, sign, etc